.travis.yml

```diff
@@ -23,13 +23,13 @@ matrix:
         - pip install mkdocs python-markdown-math --user
         - PATH=$PATH:~/.local/bin mkdocs build
         - mkdir build && pushd build
-        - cmake -DCMAKE_INSTALL_PREFIX=~/.local -DWITH_CXX=ON ..
+        - cmake -DCMAKE_INSTALL_PREFIX=~/.local -DNLOPT_WITH_CXX=ON ..
         - make install -j2 && ctest -j2 --output-on-failure
         - rm -rf * ~/.local
-        - cmake -DCMAKE_INSTALL_PREFIX=~/.local -DWITH_CXX=ON -DBUILD_PYTHON=OFF -DBUILD_OCTAVE=OFF -DBUILD_GUILE=OFF -DCMAKE_TOOLCHAIN_FILE=$PWD/../cmake/toolchain-i686-w64-mingw32.cmake ..
+        - cmake -DCMAKE_INSTALL_PREFIX=~/.local -DNLOPT_WITH_CXX=ON -DNLOPT_BUILD_PYTHON=OFF -DNLOPT_BUILD_OCTAVE=OFF -DNLOPT_BUILD_GUILE=OFF -DCMAKE_TOOLCHAIN_FILE=$PWD/../cmake/toolchain-i686-w64-mingw32.cmake ..
```

Comment:
I'm fine with prefixes, but my first inclination is to prefer shorter options: `NLOPT_CXX=ON`, `NLOPT_PYTHON=ON`, `NLOPT_OCTAVE=OFF`, etcetera, unless they are conflicting with something else

Reply:
good idea